src/styles/blocks/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__information {
+    margin-top: 40px;
+  }
+
+  &__title {
+    color: $primary-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary-color;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &-stars {
+      display: flex;
+    }
+
+    &-total {
+      color: $primary-color;
+      text-align: right;
+    }
+  }
```

Comment:
This is so incorrect😢
1. Star should be a separate BEM block and have its own file
2. Just because names of elements have a common word, doesn't mean you need to apply `&` to them. It should be 
```
.card {
  &__review {}
  &__review-start {}
  &__review-total {}
}